src/commands/base.mjs

```diff
@@ -24,14 +25,25 @@ export class BaseCommand extends ShellRunner {
     if (!chartReleaseName) throw new MissingArgumentError('chart release name is required')
 
     if (chartDir) {
-      const chartPath = `${chartDir}/${chartReleaseName}`
-      await this.helm.dependency('update', chartPath)
```

Comment:
are we no longer doing a `helm dependency update` or is it being called somewhere else?

Reply:
Good question. I am going to add it back. For me it was repeatedly updating the chart for every command which was annoying.

Reply:
It might be useful to have an option to disable it, maybe you are wanting to use an older version or make some changes locally, or make it run faster for development as in your case.